constructor/train.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import pandas as pd
+import io
+import constructor as ctor
+import psycopg2
+import json
+import torch
+import structures
+import random
+from psycopg2 import sql
+
+curr_id = int(sys.argv[1])
+
+con = psycopg2.connect(
+  database="nnbuilder",
+  user="nnbuilder",
+  password="nnbuilder",
+  host="127.0.0.1",
+  port="5432"
+)
+
+cur = con.cursor()
+
+cur.execute("SELECT * from tasksqueue where task_id=%s", [curr_id])
+row = cur.fetchone()
+task_info = structures.Task(row)
+
+cur.execute("SELECT * from datasets where ds_id=%s", [task_info.dataset])
+row = cur.fetchone()
+dataset = structures.Dataset(row)
+cur.execute("SELECT * from neuralnetworks where nn_id=%s", [task_info.nnDescription])
+row = cur.fetchone()
+
+info = json.loads(row[1])
+model = ctor.NeuralNetwork(info, task_info.task_type)
+
+if task_info.model is not None:
```

Comment:
What happens if it's none?

Reply:
This means that the model has not been trained before and the default states